.github/workflows/archery.yml

```diff
@@ -48,13 +50,15 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v1
         with:
-          python-version: '3.7'
-      - name: Install
+          python-version: '3.5'
```

Comment:
Are you sure? That version is starting to get old...

Reply:
On older linux distros this is the default, so I was trying to make it more convenient for non-python developers. We can of course increase the python requirement later.